devel/libmcfp/Portfile

```diff
@@ -22,3 +22,8 @@ installs_libs       no
 checksums           rmd160  23fb5965d4431f4d2df89562de137db97b2bb873 \
                     sha256  d35e83e660c3cb443d20246fea39e78d2a11faebe3205ab838614f0280c308d0 \
                     size    23580
+
+# Your GNU g++ is too old, need at least 9.4
+# Undefined symbols: "__ZN5Catch29formatReconstructedExpressionERSoRKNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEENS_9StringRefES8_"
+compiler.cxx_standard \
+                    2011
```

Comment:
thanks for the fix - upstream actually lists "2017" as the minimum requirement so please update it to that value.

Reply:
Done